doc/nrf/ug_matter_overview_network_topologies.rst

```diff
@@ -0,0 +1,147 @@
+  The devices in a fabric share the same :ref:`root of trust <ug_matter_network_topologies_concepts_security>` and configuration state and are identifiable with a unique 64-bit Fabric ID.
+  Using the Multiple Fabrics feature, you can assign nodes to a specific set of fabrics.
+
+Multi-admin
+  A Matter feature that allows to connect Matter devices to multiple applications and ecosystems out of the box.
+
+Node
+  A single instance of a Matter device, available on an IPv6 network after commissioning with a Node ID and security credentials.
+  A node can be commissioned to multiple separate fabrics and have multiple administrators using the Multiple Fabrics feature, and thus have multiple Node IDs.
+  Nodes in Matter support home automation bridging, which allows to implement other home automation technologies, transports, and link layers.
+  Each node has assigned a unique 64-bit Node ID, a 16-bit Vendor ID (VID), statically allocated by the CSA, a 16-bit Product ID (PID), and a 16-bit Group ID (GID), which identifies a set of nodes across a fabric.
+
+OTA Provider
+  A Matter node that can respond to queries from the OTA Requestor node about available software update images and provide OTA Requestor with the update packages during the Over-the-Air (OTA) software update process.
+
+OTA Requestor
+  A Matter node that can request information about available software update images to the OTA Provider node and receive update packages from it during the OTA software update process.
+  An OTA Requestor can also handle announcements about the available OTA Providers in the Matter network.
+
+.. _ug_matter_network_topologies_concepts_security:
+
+Matter network security
+***********************
+
+The Matter network security aims at authenticating only trustworthy devices to the Matter fabric and protecting the confidentiality of messages exchanged between the fabric nodes.
+
+Device authentication
+=====================
+
+Root of trust is a concept within Matter that is centered around a certification authority (CA), identified by Root Public Key (Root PK).
+The CA is a device tasked with issuing and assigning Node Operational Credentials (NOCs).
+NOCs are used to identify a node within a fabric and are protected by the Root Private Key.
+This process takes place during the security setup with Certificate-Authenticated Session Establishment step.
+NOCs are installed during the :ref:`ug_matter_network_topologies_commissioning` by the commissioner together with Trusted Root CA Certificates.
+
+Both nodes and fabrics are referenced with tuples consisting of the Root PK and the Node ID (for nodes) or Fabric ID (for fabrics).
+
+Message confidentiality
+=======================
+
+The Matter protocol uses elliptic curve cryptography as the principal mean of both public and private key protection and for providing digital signatures.
+The elliptic curve cryptography is based on the NIST P-256 curve (secp256r1).
```

Comment:
I have a feeling info about SPAKE2+ should be moved to "Device authentication" section as it's a protocol parallel to SIGMA that it a basis of CASE described in that section. The this section could only inorm about AES-CCM mode used for encrypting exchanged packets.
We can do this as a followup though.